kevm-pyk/src/kevm_pyk/kproj/evm-semantics/state-utils.md

```diff
@@ -479,6 +487,7 @@ The `"rlp"` key loads the block information.
          => #if ( notBool Ghasbasefee << SCHED >> )
                 orBool TXTYPE ==K Legacy
                 orBool TXTYPE ==K AccessList
+                orBool TXTYPE ==K Blob
```

Comment:
I'm just unsure about this part. Do you have any thoughts, @anvacaru? Please see this [Gas Accounting](https://eips.ethereum.org/EIPS/eip-4844#gas-accounting) section of EIP-4844 for context.

Reply:
This seems wrong. Blob transactions are newer than DynamicFee transactions and should have the fields of all prior types, including the DynamicFee fields.